hnn_core/params_default.py

```diff
@@ -440,6 +440,7 @@ def _secs_L2Pyr():
 
 def _secs_L5Pyr():
     """The geometry of the default sections in L5Pyr Neuron."""
+    # Neuron shape based on Jones et al., 2009
```

Comment:
Is this necessary? I also thought this was originally specified in Jones 2007, but I could be wrong.